bundles/org.openhab.binding.bmwconnecteddrive/src/main/java/org/openhab/binding/bmwconnecteddrive/internal/handler/VehicleChannelHandler.java

```diff
@@ -334,7 +336,11 @@ protected void updateLastTrip(LastTrip trip) {
     }
 
     protected void updateChargeProfileFromContent(String content) {
-        ChargeProfileWrapper.fromJson(content).ifPresent(this::updateChargeProfile);
+        try {
+            ChargeProfileWrapper.fromJson(content).ifPresent(this::updateChargeProfile);
```

Comment:
ChargeProfileWrapper.fromJson does not throw any Exception. It will return empty Optional in case the Json is unparseable.

Reply:
I see